ref/general/csec-LDAP-registry.adoc

```diff
@@ -16,67 +16,40 @@
+LDAP is an open protocol that supports authentication and authorization services.
+LDAP specifies the language that application servers use to communicate with the user registries that store security information such as usernames, passwords, and user groups.
+Although a link:/docs/ref/general/#csec-basic-registry.html[basic user registry] might be sufficient during development and testing of an application, an LDAP user registry provides the extra security that is required for applications in production.
 
+You can configure Open Liberty to retrieve security information from an LDAP server if you are building an application that uses a preexisting user registry.
+LDAP is the industry standard for directory servers, providing a means to create, modify, search, and delete users and groups in your applications.
+There are many LDAP servers available, such as the link:https://www.ibm.com/support/knowledgecenter/SSVJJU_6.4.0/com.ibm.IBMDS.doc_6.4/c_po_SDS_overview.html[IBM Security Directory Server] and the Microsoft Active Directory.
+There are also open source LDAP servers, such as link:http://www.openldap.org/[OpenLDAP] and link:http://directory.apache.org/apacheds/[ApacheDS].
 
 == Federated LDAP registries
-LDAP registries are federated by default. If you configure more than one LDAP registry in your `server.xml` file, then the registries are automatically federated with no additional configuration required. LDAP registries can be federated with one another or with custom and basic user registries. For more information, see link:/docs/ref/general/#csec-federated-registries.html[Federated user registries].
-
-== LDAP registries and the Social Media Login feature
-You can use Open Liberty's Social Media Login feature to help manage authentication with a configured LDAP registry. Depending how you configure the feature, users can either supply their social media credentials to authenticate to the LDAP registry or choose between their LDAP sign in and social media credentials for authentication. For more information, see the example configurations for the link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login feature].
 
+You can combine multiple user registries into a single federated registry to manage authentication and authorization.
+Federated registries provide a single logical view of multiple user registries and repositories.
 
-== LDAP certificate map mode
-You can use certificate map mode to map X.509 public key certificates into an external LDAP user registry for authentication. You can choose from four different map modes to specify how a certificate is mapped to the LDAP registry.
+In cases where user and group information is spread across multiple registries, federation enables Open Liberty to use this distributed user information in a unified manner with continuous store of information.
```

Comment:
For "with continuous store of information", I think there may be a typo as it reads a bit awkwardly. If so, maybe change it to "with a continuous store of information".